hack/runtime/default_mizar_network_subnet_template.json

```diff
@@ -0,0 +1,14 @@
+{
+   "apiVersion": "mizar.com/v1",
+   "kind": "Subnet",
+   "metadata": {
+      "name": "{{.Tenant}}-default-network-subnet"
+   },
+   "spec": {
+      "ip": "21.0.21.0",
+      "prefix": "24",
+      "bouncers": 3,
```

Comment:
for now 1 seems more proper

Reply:
change to 1 right now. Use 3 to test whether multi-bouncers can be automatically created.